internal/tiltfile/config/resources.go

```diff
@@ -44,8 +45,20 @@ func setEnabledResources(thread *starlark.Thread, fn *starlark.Builtin, args sta
 	return starlark.None, nil
 }
 
+func clearEnabledResources(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	err := starkit.SetState(thread, func(settings Settings) Settings {
+		settings.disableAll = true
```

Comment:
should this also set enabledResources to nil? i guess it doesn't really matter.

Reply:
Yeah, shouldn't affect functionality at the moment, but probably better to have it, to make surprises less likely when changing related code.